app/controllers/idv/review_controller.rb

```diff
@@ -26,11 +26,23 @@ def new
 
     def create
       resolution = start_idv_session
+      track_idv_event(resolution)
       process_resolution(resolution)
     end
 
     private
 
+    def track_idv_event(resolution)
+      result = {
+        success: resolution.success,
+        idv_attempts_exceeded: idv_attempter.exceeded?,
+        kbv: FeatureManagement.proofing_requires_kbv?,
+        new_phone_added: phone_confirmation_required?
```

Comment:
this tracks whether phone confirmation is required, but not whether it succeeded or not.

i.e. IdV is not complete until the phone is confirmed, so maybe we need a second analytics call after that step.